blockly-ws-search/src/css.js

```diff
@@ -1,25 +1,16 @@
 /**
  * @license
  * Copyright 2020 Google LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * SPDX-License-Identifier: Apache-2.0
  */
 
 /**
  * @fileoverview Styling for workspace search.
  * @author kozbial@google.com (Monica Kozbial)
  */
 
+import * as Blockly from 'blockly';
+
```

Comment:
Import only 'blockly/core' 
As a library unless you need access to Blocks and the JavaScript generator, it's best to only include core.